src/shared/components/annotation/genomeNexus/MutationAssessor.tsx

```diff
@@ -19,7 +19,8 @@ export default class MutationAssessor extends React.Component<
     IMutationAssessorProps,
     {}
 > {
-    static MUTATION_ASSESSOR_URL: string = 'http://mutationassessor.org/r3/';
+    // Replace to new url when manuscript available
```

Comment:
This is a `TODO`, right?

```suggestion
    // TODO Replace to new url when manuscript available
```